changelog/generated/v1.6.0.md

```diff
@@ -0,0 +1,50 @@
+## v1.6.0
+
+### Additions
+
+- For Golang-based operators, added the `declarative.go/v1` plugin which customizes initialized projects with patterns from [kubernetes-sigs/kubebuilder-declarative-pattern](https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern). (e.g `operator-sdk create api --plugins=go/v3,declarative`). ([#4731](https://github.com/operator-framework/operator-sdk/pull/4731))
+- Added `kustomize.common/v1` plugin which scaffolds the a commonly used project base that leverages `kustomize`. ([#4730](https://github.com/operator-framework/operator-sdk/pull/4730))
+- (ansible/v1,helm/v1) Add componentConfig option. For further information check [here](https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html). ([#4701](https://github.com/operator-framework/operator-sdk/pull/4701))
+- (ansible/v1,helm/v1) Add rules for leader election. ([#4701](https://github.com/operator-framework/operator-sdk/pull/4701))
```

Comment:
Nit, since this sounded confusing. I think this reflects the change
```suggestion
- (ansible/v1,helm/v1) Added rules for leader election resources in `leader_election_role.yaml`. ([#4701](https://github.com/operator-framework/operator-sdk/pull/4701))
```